src/util/rpc/http/json_rpc_http_server.cpp

```diff
@@ -177,19 +177,25 @@ namespace cbdc::rpc {
             params = req["params"];
         }
 
+        uint64_t id = 0;
```

Comment:
These changes are introduced to echo the request ID back in the response, in stead of defaulting to 0. Even though our current request pattern disallows responses and requests to be out-of-order, it's still sensible to have the ID echoed back - it is the convention for json-rpc.